nautobot/ipam/migrations/0004_fixup_p2p_broadcast.py

```diff
@@ -0,0 +1,48 @@
+from django.db import migrations
+
+import netaddr
+
+
+def fixup_p2p_broadcast(apps, schema_editor):
+    """Correct the "broadcast" field for /31 and /127, as per https://github.com/nautobot/nautobot/pull/509."""
+    Aggregate = apps.get_model("ipam", "Aggregate")
+    Prefix = apps.get_model("ipam", "Prefix")
+    IPAddress = apps.get_model("ipam", "IPAddress")
```

Comment:
Not a huge deal, so I'll leave it up to you, but the `get_model()` also accepts the dot-notation e.g. `get_model("ipam.Aggregate")`, which is the style we've used in previous data migrations.

Reply:
Good to know. Since either approach works, I'll leave this be. Thanks!